.drone.yml

```diff
@@ -21,13 +21,13 @@ workspace:
 matrix:
   PHP_VERSION:
     - 5.6
-    - 7.1
+    # - 7.1
```

Comment:
This file need to be removed or aligned with our drone. Feel free to remove NE Platform 3 builds and include tests to all new drush commands in pipeline (diffy and module eval)

Reply:
Done, but for some reason now GIT is not able to checkout the pull request any longer :(
https://drone.fpfis.eu/ec-europa/toolkit/1178/9

Reply:
Maybe you only need to sync back 3.0.22 into your branch.